proto/streamlit/proto/Element.proto

```diff
@@ -76,7 +80,7 @@ message Element {
     TimeInput time_input = 26;
     VegaLiteChart vega_lite_chart = 10;
     Video video = 14;
-    NumberInput number_input = 29;
+    NumberInput number_input = 31;
```

Comment:
Don't renumber fields. This should stay as `29`, and the new ones should be `30` and `31`.